frame/vesting/src/lib.rs

```diff
@@ -649,6 +654,25 @@ impl<T: Config> Pallet<T> {
 
 		Ok((schedules, locked_now))
 	}
+
+	#[cfg(any(feature = "try-runtime", feature = "fuzzing", test, debug_assertions))]
+	pub fn do_try_state() -> Result<(), &'static str> {
```

Comment:
ditto.

Reply:
@Doordashcon please mark comments resolved once you've addressed them